LeapEngine/SceneGraph/GameObject.cpp

```diff
@@ -2,12 +2,16 @@
 
 #include "../Components/Transform/Transform.h"
 
-#include "Debug.h"
 #include "SceneManager.h"
 
 leap::GameObject::GameObject(const char* name)
 	: m_Name{ name }
 {
+	if (TransformComponentID == -1)
```

Comment:
TransformComponentID doesn't need to be initialized in a method and can be initialized in the inline declaration or in the cpp file